pykechain/models/widgets/widgets_manager.py

```diff
@@ -786,11 +793,11 @@ def add_service_widget(
         service_id = check_base(service, Service, "service", method=self._client.service)
 
         meta.update({
-            "serviceId": service_id,
-            "emphasizeButton": emphasize_run,
-            "showDownloadLog": check_type(download_log, bool, "download_log"),
-            "showLog": True if download_log else check_type(show_log, bool, "show_log"),
-            "alignment": check_enum(alignment, Alignment, "alignment"),
+            AssociatedObjectId.SERVICE_ID: service_id,
+            MetaWidget.EMPHASIZE_BUTTON: emphasize_run,
+            MetaWidget.SHOW_DOWNLOAD_LOG: check_type(download_log, bool, "download_log"),
+            MetaWidget.SHOW_LOG: True if download_log else check_type(show_log, bool, "show_log"),
+            MetaWidget.ALIGNMENT: check_enum(alignment, Alignment, MetaWidget.ALIGNMENT),
```

Comment:
The string in the final input for the `check____()` methods is the keyword of the Pykechain function/method, not the KE-chain meta, although they might overlap.

Reply:
Reverted bacl to 'alignment'